src/cli/managers/config_manager.py

```diff
@@ -24,5 +23,40 @@ def get_applications(self) -> List[Dict[str, str]]:
     def application_exists(self, application: str) -> bool:
         return True
 
+    def remote_application_exists(self, application: str) -> Tuple[bool, int]:
+        """
+        Check if the application is already created on remote
+        by checking the remote_id field in the config file.
+        Returns True and application id if application exists on remote
+        Returns False and -1 if application does not exist on remote
+        """
+        application_info = self.get_application(application)
+        if 'remote_id' in application_info:
+            return True, int(application_info['remote_id'])
+
+        return False, -1
```

Comment:
The tuple contains redundant information. When remote_id >= 1 the result is True.
Suggestion: why not only return remote_id or -1 as in `return application_info.get('remote_id', -1)`